convert.html

```diff
@@ -28,6 +28,14 @@
 </head>
 <body style='color:white'>
 
+<script>
+
+if (window.location.hostname.indexOf("vdo.ninja") == 0) {

```

Comment:
Here, [eqeqeeq ](https://eslint.org/docs/rules/eqeqeq)should apply

```suggestion
if (window.location.hostname.indexOf("vdo.ninja") === 0) {

```